nebula/ui/components/VPNServerList.qml

```diff
@@ -128,19 +125,31 @@ FocusScope {
 
             VPNRecentConnections {
                 id: recentConnections
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.rightMargin: VPNTheme.theme.windowMargin / 2
-                anchors.leftMargin: anchors.rightMargin
+                Layout.fillWidth: true
+                Layout.rightMargin: VPNTheme.theme.windowMargin / 2
+                Layout.leftMargin: VPNTheme.theme.windowMargin / 2
+                Layout.preferredHeight: showRecentConnections ? implicitHeight : 0
                 visible: showRecentConnections && serverSearchInput.text.length === 0
                 showMultiHopRecentConnections: false
-                height: showRecentConnections ? implicitHeight : 0
             }
 
             Repeater {
                 id: countriesRepeater
                 model: countriesModel
-                delegate: VPNServerCountry{}
+                delegate: VPNServerCountry{
+                    //Necessary because VPNClickableRow has hardcoded anchors (that we aren't using here because this is a layout)
+                    anchors.left: undefined
+                    anchors.right: undefined
+                    anchors.leftMargin: undefined
+                    anchors.rightMargin: undefined
```

Comment:
Nit: For setting multiple `anchors` properties I really like using:
```suggestion
                    anchors {
                        left: undefined
                        right: undefined
                        leftMargin: undefined
                        rightMargin: undefined
                    }
```

Reply:
I have no preference either way - but I know this is whats recommended by Qt's own conventions. My vote is for consistency. If we can get buy-in from all UI engineering to do this going forward, I am for it. 

Maybe we should start a document on our own qml coding conventions?